src/components/menu/editor-mode-title.jsx

```diff
@@ -23,24 +42,17 @@ export class TitleUnconnected extends React.Component {
 
   render() {
     const elem = (
-      <div className="title-field">
-        <div
-          className={`title-field-contents ${
-            this.props.additionalContainerClasses
-          }`}
-        >
-          <Helmet title={this.getTitle()} />
-
-          <input
-            className={`page-title ${this.props.titleInputClasses}`}
-            value={this.props.title || ""}
-            placeholder="new notebook"
-            onChange={evt => {
-              tasks.changeTitle.callback(evt.target.value);
-            }}
-          />
-        </div>
-      </div>
+      <TitleContainer>
+        <Helmet title={this.getTitle()} />
+        <InputElement
+          isUntitled={this.props.isUntitled}
+          value={this.props.title || ""}
+          placeholder="new notebook"
+          onChange={evt => {
+            tasks.changeTitle.callback(evt.target.value);
```

Comment:
we should just import the action directly (task is not needed) and pass in with mapDispTP